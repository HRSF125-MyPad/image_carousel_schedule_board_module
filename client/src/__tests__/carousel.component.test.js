import React from 'react';
import renderer from 'react-test-renderer';
import Carousel from '../carousel.jsx';

describe('Carousel', () => {
    it ('renders correctly', () => {
        const test = renderer.create(
            <Carousel/>
        ).toJSON();
        expect(test).toMatchSnapshot();
    })
})