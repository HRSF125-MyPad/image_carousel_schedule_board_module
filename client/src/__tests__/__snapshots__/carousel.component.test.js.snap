// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel renders correctly 1`] = `
<div>
  <div>
    hello from carousel
  </div>
</div>
`;
